doc/example/hierarchical.ipynb

```diff
@@ -0,0 +1,877 @@
+      "text/plain": [
+       "<Figure size 1500x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Waterfall plot - hierarchical optimization with analytical inner solver vs standard optimization\n",
+    "pypesto.visualize.waterfall(\n",
```

Comment:
It's because `order_by_id=True`. The different scenarios use the same starting points (for the outer parameters), so for the sake of this comparison, it's probably more informative to plot the results from the same starting point together (which may mean non-monotonous ordering for an individual scenario).